internal/db/db.go

```diff
@@ -0,0 +1,79 @@
+package db
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/jackc/pgx/v4/pgxpool"
+)
+
+type DB struct {
+	Pool *pgxpool.Pool
+}
+
+type ComposeEntry struct {
+	Id uuid.UUID
+	Request json.RawMessage
+	CreatedAt time.Time
+}
+
+// Run a local container: sudo podman run -p5432:5432 --name imagebuilder -ePOSTGRES_PASSWORD=foobar -d postgres
+func InitDBConnectionPool(connStr string) (*DB, error) {
+	dbConfig, err := pgxpool.ParseConfig(connStr)
+	if err != nil {
+		return nil, err
+	}
+
+	pool, err := pgxpool.ConnectConfig(context.Background(), dbConfig)
+	if err != nil {
+		return nil, err
+	}
+
+	return &DB{pool}, nil
+}
+
+func (db *DB) InsertCompose(uuid string, request json.RawMessage, accountId, orgId string) error {
+	ctx := context.Background()
+	conn, err := db.Pool.Acquire(ctx)
+	if err != nil {
+		return err
+	}
+	defer conn.Release()
+
+	_, err = conn.Query(ctx, "INSERT INTO composes(id, request, created_at, account_id, org_id) VALUES ($1, $2, CURRENT_TIMESTAMP, $3, $4);", uuid, request, accountId, orgId)
```

Comment:
I stumbled on the `created_at` as it repeats data that should be in `composer`. However, I realize that we (will probably) need this in order paginate, and not having to query composer for all composes all the time, but only the ones we want to show on screen. So: all good! (this was a vacuous comment)

Reply:
yea I mean created_at maybe shld be inserted_at, that could be more correct